data/registry_model/registry_proxy_model.py

```diff
@@ -446,6 +492,18 @@ def _create_manifest_with_temp_tag(
         manifest: ManifestInterface,
         manifest_ref: str | None = None,
     ) -> tuple[Manifest | None, Tag | None]:
+        """
+        Creates a manifest in the given repository. Also creates placeholders for the
+        objects referenced by the manifest. For manifest lists, it creates
+        sub manifests entries attached to the manifest list along with a temporary tag.
+
+        Also checks whether the given manifest size is within the quota limit
+        of the namespace the repository is part of. If not, it prunes older tags.
+        Raises QuotaExceededException if there are not enough tags to prune.
+        """
+        # check if the given manifest size is within quota limit
+        self._check_blob_upload_and_prune(repository_ref, manifest)
```

Comment:
is there a way to communicate what your comment is saying by improving the function name?

Reply:
that's better!